bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/client/SerialClient.java

```diff
@@ -69,7 +69,8 @@ protected KNXNetworkLink establishConnection() throws KNXException, InterruptedE
                     e);
         } catch (KNXException e) {
             final String msg = e.getMessage();
-            if ((msg != null) && (msg.startsWith("can not open serial port"))) {
+            // TODO add a test for this string match; error message might change in later version of Calimero library
+            if ((msg != null) && (msg.startsWith("failed to open serial port"))) {
```

Comment:
You could at least define a constant?

Reply:
Done as requested. This was not my code. I tried to keep the change minimal and just adapted to the new message.